src/django/api/views/v1/moderation_events.py

```diff
@@ -47,3 +51,34 @@ def list(self, request):
             query_body
         )
         return Response(response)
+
+    @handle_errors_decorator
+    def retrieve(self, request,  pk=None):
+        if not IsRegisteredAndConfirmed().has_permission(request, self):
+            return Response(
+                {
+                    "detail": (
+                        "Only an authorized user "
+                        "can perform this action."
+                    )
+                },
+                status=status.HTTP_401_UNAUTHORIZED
+            )
```

Comment:
I suppose the moderation events should be accessible only to moderators.

Reply:
@mazursasha1990 No, there is an requirement that authorized user who want to check the status of his request (moderation event) can get this data by the GET request. As when user publish the POST/PATCH request to moderation queue he get moderation_id (uuid) by this id user could track status.

Reply:
Okay.
Thank you.